src/components/help-icon/help-icon.test.jsx

```diff
@@ -10,6 +10,11 @@ describe('HelpIcon', () => {
     text: 'Help Text',
   };
 
+  // There needs to be something on the DOM that the tooltip is describing, so render a dummy div for each test.
+  beforeEach(() => {
```

Comment:
A more standardized pattern for "setting up the env for a given test suite" is the `beforeAll`. And a similar clean up in a `afterAll`. I think it's more of a convention. Normally, it wouldn't work for Mavenlink (tm) tests because we are accustomed to cleaning up the whole env between each test due to global pollution.

Reply:
I think in this case it doesn't work as well. Since `cleanup` is called `afterEach` test, and `beforeAll` only happens once, only the first test will pass. Subsequent tests will fail because this div has been unmounted by `cleanup`.

There's a case to be made that since we're calling `render` twice there's already some weirdness, but this is definitely an edge case which was caused by the `useLayoutEffect` added

![image](https://user-images.githubusercontent.com/3209944/124647000-9d240480-de52-11eb-8101-84dff549a784.png)

Reply:
Oh, right, the auto-cleanup. 😓 

I guess the other route is injecting a native DOM node yourself. But then we don't get the auto-cleanup. 

/shrug I'd stick with as-is.